packages/@react-types/table/package.json

```diff
@@ -9,6 +9,7 @@
     "url": "https://github.com/adobe/react-spectrum"
   },
   "dependencies": {
+    "@react-spectrum/dnd": "^3.0.1",
```

Comment:
Hmm not ideal to have a react-spectrum dependency in react-types... Wonder if we should move `SpectrumTableProps` into the `@react-spectrum/table` package also?